### ISSUE TITLE
jupyter notebook

### ISSUE THREAD

**Issue by ozanhalisilter** [carykh/jumpcutter#170]
Is it possible to use this in jupyter notebook or google colab?

**Comment by CommanderStorm** [comment]
Its best for jupyter progrmms to execute fast.
This programm takes ages and is such not a good usecase.

just DONT

**Comment by jimmyli56** [comment]
yeah, it takes 10+ minutes to process anything big as it processes the video frame by frame. which is why it takes so long.
